packages/typography/src/TypographySubcomponents.test.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
_Just a thought_: This is more like an integration test, and we haven't set any rules for where to put, or how to name these. Lets leave it be for now.

Reply:
Let's be real, this is a cheap test coverage grab.